app/assets/javascripts/admin/addon/services/admin-sidebar-state-manager.js

```diff
@@ -57,6 +58,10 @@ export default class AdminSidebarStateManager extends Service {
     this.sidebarState.isForcingSidebar = false;
   }
 
```

Comment:
I need it because importing when I import `admin/components/modal/admin-search` in `admin-sidebar.js` it was failing for non admin users.